src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -1,10 +1,10 @@
 ---
 title: "Session replay"
```

Comment:
@ally-sassman this is outside the scope of your current changes, but I'm not seeing a lot of "FAQ" headings in our docs. At some point, you might make merge that into "How session replay works" and just introduce them as questions about how the feature affects their environment.

Reply:
I posted something in the doc-talk about trying out a mini faq section for this LP doc, so I'll wait to see what Shawn and Austin think before removing/modifying it.